app/src/main/java/com/alphawallet/app/service/AssetDefinitionService.java

```diff
@@ -781,20 +819,25 @@ private void checkDownloadedFiles()
                 .subscribe(this::handleFileLoad, this::onError).isDisposed();
     }
 
-    private Single<File> updateSignature(File file)
+    /* Add cached signature if uncached files found. */
+    private Single<File> cacheSignature(File file)
     {
+        // note that outdated cache is never deleted - we don't have that level of finesse
```

Comment:
Recommend capturing memory bloat in an issue and referencing here

Reply:
the cache is not in the memory btw. they are on mobile storage.